.github/workflows/all-tests.yml

```diff
@@ -162,7 +162,7 @@ jobs:
     strategy:
       matrix:
         java-version: [8, 11]
-        bitbucket-version: [6.7.1, 7.6.0]
+        bitbucket-version: [6.7.1, 8.0.0-breakit-m13]
```

Comment:
@mgoyal2-atl - I don't believe this code will actually work as expected on `6.7.1`. From [the API changelog for 7.2](https://developer.atlassian.com/server/bitbucket/reference/api-changelog/#tasks):

> `BLOCKER` comments have existed since 6.7, however, _prior to 7.0 they are not treated as tasks._ Apps that wish to retain semantic compatibility with 6.x must continue using the `TaskService` until support for pre-7.0 releases can be dropped.

As a result, maybe the earliest version should be 7.0 or (since that's out of support) 7.6.0.